### DISCUSSION TITLE
v5 - Share your CSS tweaks here!

### DISCUSSION THREAD

**Discussion by ntevenhere** [mbnuqw/sidebery#1254] [Show and tell]
Currently I don't have any because to me the Proton theme *seems* perfect, but I'm curious if it could be improved. I'm curious what are your usage needs that require CSS tweaks.

**Comment by Elementari** [comment]
```
#root.root {--tabs-activated-shadow: 0 1px 4px -1px rgba(0,0,0,0.259), inset 0 0 0 0px #146fd9;}
#root.root {--tabs-margin: 0px;}
#root.root {--tabs-border-radius: 0px;}

#root.root {--ctx-menu-icon-opt-height: 30px;}
#root.root {--ctx-menu-text-opt-height: 20px;}
#root.root {--ctx-menu-text-opt-inner-gap: 3px;}
#root.root {--ctx-menu-font: 0.82rem sans-serif;}


#root.root {--tabs-indent: 20px;}
#root.root {--tabs-inner-gap: 2px;}

#root.root {--tabs-activated-fg: #000000ff;}

#root.root {--nav-delimiter-size: 12px;}
#root.root {--nav-btn-width: 28px;}
#root.root {--search-icon-width: 26px;}
#root.root {--search-font-size: 0.80rem;}
#root.root {--ntb-btn-height: 20px;}
#root.root {--tabs-activated-bg: #f6cbbbff;}

#root.root {--nav-btn-height: 22px;}
#root.root {--tabs-font: .7rem sans-serif;}
#root.root {--tabs-height: 17px;}
#root.root {--tabs-pinned-height: 22px;}
#root.root {--tabs-pinned-width: 30px;}

#root.root {--search-height: 18px;}
```

_________

these are all changes made with the style editor interface, no additional manual changes. 
the color of the active tab is the same as my FF theme, so you may want to change that.
it's used on a MBA 13" retina. 
I use only one indent level, and I usually have hundreds of tabs.

**Reply by ntevenhere** [reply]
Compact everything, nice!
![image](https://github.com/mbnuqw/sidebery/assets/66701832/ecf880f1-88e8-4f22-92ad-3a266514132a)

**Comment by Elementari** [comment]
FWIW, the font size in general settings I have set to XXS, obviously one can tweak this and #root.root {--tabs-font: .7rem sans-serif;} ; I personally like it this way; XS just increases the distance between letters, making it more (!) difficult to read for me actually (the eye can grasp a word without movement when letters are close together; wide spacing is only relevant if you need to distinquish individualy letters with high accuracy).

the attached picture has been slightly manipulated by me (cut out some tabs). I.e. the small glitches are not there actually.
<img width="432" alt="sideberry" src="https://github.com/mbnuqw/sidebery/assets/33958589/b3eb0af5-7ccc-4640-8557-39d331941043">

**Comment by Elementari** [comment]
search panel can be toggled by setting to dynamic and adding search icon to navigation bar.
sorry for the mess in that bar; forgot to fix this:
setup is now: 

default panel, create panel, hidden panels, space--, mute, collapse all, history, bookmarks, snapshots, settings, search icon. 

- I have removed all bars at the bottom of the tabs tree
- new tab is double click into gray area
- doublicate tab is double click onto tab.
- click on panel icon scrolls up / down all the way
